client/src/components/admin/SurveyListItem.tsx

```diff
@@ -65,20 +66,21 @@ export default function SurveyListItem(props: Props) {
     }/${survey.name}`;
   }, [survey.name]);
 
+  const cardStyle = survey.isPublished
```

Comment:
Inline styles are usually bad practice, unless it's the only choice. In this case we can rely on MUI's theming and add a new class inside `makeStyles`, e.g. `publishedCard: { ... }`.

The easiest way to apply multiple conditional class names is to use the `clsx` utility library. It should already be installed as a 3rd party dependency, but feel free to `npm install` it to this project as well:
```ts
import clsx from 'clsx';

clsx(loading && classes.loading, survey.isPublished && classes.publishedCard)
```